crates/nu-cli/src/syntax_highlight.rs

```diff
@@ -141,6 +96,8 @@ impl Highlighter for NuHighlighter {
     }
 }
 
+// Splits span into sub-spans.  Tuple booleans determines, wherever the span
+// should be highlighted.
```

Comment:
If you use `///` comments here the editor hovers will be nicer :)

```suggestion
/// Splits span into sub-spans.  
/// Tuple booleans determines, wherever the span should be highlighted.
```